### PR TITLE
Add pcl aport

### PR DESCRIPTION
**Pull request by chambbj**

### CODE REVIEW

File: testing/pcl/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+pkgname=pcl
+pkgver=1.8.1
+pkgrel=0
+pkgdesc="Point Cloud Library (PCL)"
+url="https://github.com/PointCloudLibrary/pcl"
+arch="all"
+license="BSD"
+options="!check"
+depends=""
+makedepends="cmake eigen-dev boost-dev flann-dev"
```
Comment (BernhardGruen) [inline testing/pcl/APKBUILD]: It seems this package needs `flann-dev` which is not yet in aports.

File: testing/pcl/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# Contributor: Bradley J Chambers <brad.chambers@gmail.com>
+# Maintainer: Bradley J Chambers <brad.chambers@gmail.com>
+pkgname=pcl
+pkgver=1.8.1
+pkgrel=0
+pkgdesc="Point Cloud Library (PCL)"
+url="https://github.com/PointCloudLibrary/pcl"
+arch="all"
+license="BSD"
```
Comment (BernhardGruen) [inline testing/pcl/APKBUILD]: please see https://spdx.org/licenses/

File: testing/pcl/APKBUILD (hunk 1 above)
Comment (clandmeter) [reply testing/pcl/APKBUILD]: To make travis happy you can combine commits into a single PR.

Comment (chambbj) [reply testing/pcl/APKBUILD]: @clandmeter only problem is that the pcl build takes quite a while and will timeout if combined. This is how I originally had it structured.

**Comment by chambbj** [comment]
@clandmeter As discussed on IRC today, tests have been enabled (with some exceptions, see comments). This does build with all enabled tests passing locally (running `abuild rootbld`).

**Pull request closed without merging**